CHANGELOG.md

```diff
@@ -1,24 +1,36 @@
 # Change log for Microsoft365DSC
 
+
+# UNRELEASED
+* IntuneWindowsAutopilotDeploymentProfileAzureADJoined
+  * Initial release
+    FIXES [#2605](https://github.com/microsoft/Microsoft365DSC/issues/2605)
+* IntuneWindowsAutopilotDeploymentProfileAzureADHybridJoined
+  * Initial release
+    FIXES [#2605](https://github.com/microsoft/Microsoft365DSC/issues/2605)
+* DRG
+  * Various fixes
+    Cleanup generated code
+    Fix AdditionalProperties complex constructor
+    Fix Read privileges in settings file
+
 # 1.23.322.1
 
-* AADRoleSetting
-  * Added CertificateThumbPrint and ApplicationId to the output of the Get method
-* EXODistributionGroup
-  * Fixed an error where the name wasn't properly escaped in the Filter
-    FIXES [#3044](https://github.com/microsoft/Microsoft365DSC/issues/3044)
-* IntuneDeviceConfigurationDeliveryOptimizationPolicyWindows10
-  * Initial Release
-    FIXES [#2832](https://github.com/microsoft/Microsoft365DSC/issues/2832)
 * EXORoleAssignmentPolicy
   * Fix issue with IsDefault parameter
     FIXES [#2977](https://github.com/microsoft/Microsoft365DSC/issues/2977)
+* IntuneDeviceConfigurationDeliveryOptimizationPolicyWindows10
```

Comment:
Should this be in here? Since this resource isn't included in this PR.

If it should be: I think this should be moved the the Unreleased section as well.